playground-common/playground-include-settings.gradle

```diff
@@ -72,6 +72,21 @@ def setupPlayground(settings, String relativePathToRoot) {
             new File(supportRoot, "testutils/testutils-common"))
     settings.includeProject(":internal-testutils-gradle-plugin",
             new File(supportRoot, "testutils/testutils-gradle-plugin"))
+
+    settings.gradleEnterprise {
+        server = "https://ge.androidx.dev"
+        buildScan {
+            publishAlwaysIf(System.getenv("GRADLE_ENTERPRISE_ACCESS_KEY") != null)
```

Comment:
You can accomplish the same thing with:
```
publishAlways()
publishIfAuthenticated()
```
Then this will work for other forms of authentication, such as https://docs.gradle.com/enterprise/gradle-plugin/#automated_access_key_provisioning

Reply:
I don't see documentation for this neither or https://docs.gradle.com/enterprise/gradle-plugin/#automated_access_key_provisioning nor on https://docs.gradle.com/enterprise/gradle-plugin/api/com/gradle/scan/plugin/BuildScanExtension.html . Can you point me to the docs?

Reply:
Unfortunately `publishIfAuthenticated()` is currently not documented (a hidden feature), since we weren't overly happy with the API. You can see it in use on [JUnit](https://github.com/junit-team/junit5/blob/main/settings.gradle.kts#L52-L56) and [Spring](https://github.com/spring-io/gradle-enterprise-conventions/blob/7d25b99352bdaa2e8b818fd97edf18e7b8df30b3/src/main/java/io/spring/ge/conventions/gradle/BuildScanConventions.java#L84).

Fortunately groovy lets you call the method without any reflection hoops.